apps/bfd-data-npi/src/main/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookup.java

```diff
@@ -78,7 +78,7 @@ public Optional<String> retrieveNPIOrgDisplay(Optional<String> npiNumber) {
     /*
```

Comment:
Merely holding for internal communications. We can resolve this once we're set 😉

Reply:
I think we're all set to deploy this when ready!